mypyc/genops.py

```diff
@@ -49,6 +49,21 @@ def type_to_rttype(typ: Type) -> RTType:
     assert False, '%s unsupported' % type(typ)
 
 
+class AssignmentTarget(object):
+    pass
+
+
+class AssignmentTargetRegister(AssignmentTarget):
+    def __init__(self, register: Register) -> None:
+        self.register = register
+
+
+class AssignmentTargetIndex(AssignmentTarget):
```

Comment:
Maybe rename to something like `AssignmentTargetListIndex` -- or perhaps this should eventually take the indexed object type as an argument?